lib/hanami/view/rendering/layout_scope.rb

```diff
@@ -230,7 +230,9 @@ def method_missing(m, *args, &blk)
           # that we want to be frozen in the future
           #
           # See https://github.com/hanami/view/issues/130#issuecomment-319326236
-          if @scope.respond_to?(m, true)
+          if @scope && @scope.respond_to?(m, true) && @scope.locals.has_key?(m) && layout.respond_to?(m)
```

Comment:
Is there any reason why we don't use `layout.respond_to?(m, true)`?

Reply:
there isn't, I just copy&paste from this line https://github.com/hanami/view/pull/140/files/bd867cee6c67fa9199390ee5c613071dfb321202..79d76e8af4d722a27bf2c938973134f75233f0a5#diff-5d90b9ac33da24fadae41a69b3800763R237

Reply:
Please verify that private methods work as well. Thanks 😄